src/core/processing/qgsprocessingparametertypeimpl.h

```diff
@@ -480,7 +480,7 @@ class CORE_EXPORT QgsProcessingParameterTypePoint : public QgsProcessingParamete
 
     QString description() const override
     {
-      return QCoreApplication::translate( "Processing", "A geographic point parameter." );
+      return QCoreApplication::translate( "Processing", "A point parameter." );
```

Comment:
What about 
```suggestion
      return QCoreApplication::translate( "Processing", "A point geometry parameter." );
```

Reply:
I'd thought something like about that but wasn't sure if it might collide with some meaning in Layouts. And I am not familiar enough with the terms in the modeler yet.  If you think it's better, let's use that! :)